MAINTAINERS.yml

```
@@ -1249,6 +1249,14 @@ Formatted Output:
   labels:
     - "area: Formatting Output"
 
+Google Boards:
```

"Google Platforms" would be more consistent with the existing areas we have for vendor-specific boards.

```suggestion
Google Platforms:
```